build.gradle

```diff
@@ -304,8 +309,59 @@ shadowJar {
         //  it will be at the same level as the newly created jar.  (overwriting symlink, if it exists)
         // Please note that this will cause failures in Windows, which does not support symlinks.
         def symlinkLocation = destinationDir.toString() + "/hellbender-protected.jar"
+        def symlinkLocation2 = destinationDir.toString() + "/gatk-protected.jar"
+        exec {
+            commandLine 'ln', '-fs', archivePath, symlinkLocation
+            commandLine 'ln', '-fs', archivePath, symlinkLocation2
+            ignoreExitValue = false
+        }
+    }
+}
+
+tasks.withType(ShadowJar) {
+    manifest {
```

Comment:
I think if you set the manifest in tasks.withType(jar) you will likely not need to specify it more than once